test/jdk/java/foreign/TestVarArgs.java

```diff
@@ -131,17 +130,17 @@ private static List<Arg> makeArgs(List<ParamType> paramTypes, List<StructFieldTy
         return args;
     }
 
-    private static void check(int index, MemoryAddress ptr, List<Arg> args) {
+    private static void check(int index, MemorySegment ptr, List<Arg> args) {
         Arg varArg = args.get(index);
         MemoryLayout layout = varArg.layout;
         MethodHandle getter = varArg.getter;
         List<Consumer<Object>> checks = varArg.checks;
-        try (MemorySession session = MemorySession.openConfined()) {
-            MemorySegment seg = MemorySegment.ofAddress(ptr, layout.byteSize(), session);
+        try (Arena arena = Arena.openConfined()) {
+            MemorySegment seg = MemorySegment.ofAddress(ptr.address(), layout.byteSize(), arena.scope());
             /* Vararg float is promoted to double (8 bytes) in native (See libVarArgs.c)
              * in which case float must be converted back from double at the same memory
              * address in java on the Big-Endian(BE) platforms such as AIX.
-             */
+             */ 
```

Comment:
Please remove the trailing space on this line.

Reply:
FYI: @JasonFengJ9

Reply:
Fixed.

Reply:
@ChengJin01 please make this change in your commit.

Reply:
Already did in the latest commit.